src/network/network_chat_gui.cpp

```diff
@@ -261,20 +261,55 @@ static void SendChat(const std::string &buf, DestType type, int dest)
 		NetworkServerSendChat((NetworkAction)(NETWORK_ACTION_CHAT + type), type, dest, buf, CLIENT_ID_SERVER);
 	}
 }
+class NetworkChatAutoCompletion final : public AutoCompletion {
```

Comment:
missing empty line prior to class declaration

Reply:
Done